src/components/Community/Post.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Badge from "../common/Badge";
+
+export type TeamScheme =
+  | "kia"
+  | "samsung"
+  | "lg"
+  | "doosan"
+  | "ssg"
+  | "kt"
+  | "nc"
+  | "hanhwa"
+  | "lotte"
+  | "kiwoom";
+
+export interface PostProps {
+  id: number;
+  match_date: string;
+  home_team_name: TeamScheme;
+  away_team_name: TeamScheme;
+  review_short: string;
+  user_id: number;
+  likes: number;
+  comments: number;
+}
+
+interface Props {
+  post: PostProps;
+}
+
+const Post: React.FC<Props> = ({ post }: Props) => {
```

Comment:
이제는 React.FC 타입을 잘 사용하지 않는데요!
이렇게 표현해줘도 충분할 것 같아요!
```suggestion
const Post = ({ post }: Props) => {
```